exercises/practice/binary-search/test_binary_search.zig

```diff
@@ -5,67 +5,45 @@ const binary_search = @import("binary_search.zig");
 const SearchError = binary_search.SearchError;
 
 test "finds a value in an array with one element" {
-    comptime try testing.expectEqual(0,
-        try binary_search.binarySearch(i4, 6,
-            &[_]i4{6}));
+    comptime try testing.expectEqual(0, try binary_search.binarySearch(i4, 6, &[_]i4{6}));
```

Comment:
IDK the changes in this file are the only places where I'd personally _slightly_ consider breaking up the line... But this is such a tiny style choice and I could go either way

```suggestion
    const actual = try binary_search.binarySearch(i4, 6, &[_]i4{6});
    comptime try testing.expectEqual(0, actual);
```